### ISSUE TITLE
Not compatible with the latest version of webstorm.

### ISSUE THREAD

**Issue by MarxZhou** [mohak1712/CodeSnapshot#1]
Is the plug-in out of maintenance?
Currently it cannot be used normally in v2020.3.x.

**Comment by mohak1712** [comment]
hi 
are you not able to download the plugin with the above version of webstorm ?

**Comment by MarxZhou** [comment]
> hi
> are you not able to download the plugin with the above version of webstorm ?

Yes, the current plug-in cannot be retrieved in the plug-in market of the IDE.
I can use it normally on the v2020.2.x version of the IDE.

**Comment by mohak1712** [comment]
WebStorm — 2019.3 — 2020.2.4
this is the compatible version range for now
I am checking what is the issue with the latest version

**Comment by MarxZhou** [comment]
> WebStorm — 2019.3 — 2020.2.4
> this is the compatible version range for now
> I am checking what is the issue with the latest version

Does the plug-in continue to be maintained?
